boot/bootdata/bootcd.ini

```diff
@@ -1,19 +1,28 @@
 [FREELOADER]
-DefaultOS=Setup
-TimeOut=0
+DefaultOS=LiveCD_Debug
+TimeOut=10
 
 [Display]
-TitleText=ReactOS Setup
+TitleText=ReactOS Setup & Live BootCD
 MinimalUI=Yes
 
 [Operating Systems]
 Setup="Setup"
 Setup_Debug="Setup (Debug)"
 Setup_Aacpi="Setup ACPI APIC (Debug)"
 Setup_Screen="Setup (Screen)"
+;;;;
```

Comment:
Which purpose have these semicolons? of Any comment to be added here? Or it's simply like a separator? :thinking:

Reply:
yes it was a separator, but this looks ugly indeed and I'll remove these. Maybe I'll just keep an empty line here, or just nothing.